.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Release
+on:
+  # Runs automatically when a tag beginning with 'release-' is pushed.
+  push:
+    tags:
+      - release-*
+
+jobs:
+  gh-release:
+    runs-on: ubuntu-20.04
+    env:
+      BOULDER_TOOLS_TAG: go1.17.7_2022-02-10
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: dockerhub login
+        uses: docker/login-action@v1.8.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME}}
+          password: ${{ secrets.DOCKER_PASSWORD}}
+          logout: true
+        continue-on-error: true
+
+      - name: docker-compose pull
+        run: docker-compose pull
+
+      - name: build and release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # https://cli.github.com/manual/gh_release_create
+        run: |
+          ./make-deb.sh
+          tag=$(git describe --tags --abbrev=0)
+          gh release create "${tag}" *.deb
```

Comment:
I think you can use GITHUB_REF_NAME: https://docs.github.com/en/actions/learn-github-actions/environment-variables.